cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -250,6 +250,86 @@ action.cyences_notable_event_action.products = Office 365 Defender ATP
 action.cyences_notable_event_action.teams = SOC
 
 
+[Defender ATP - System is Offboarded]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 54 * * * *
+description = This alert notifies when system/device is offboarded \
+\
+Data Collection - Defender ATP Status Check Add-on (https://splunkbase.splunk.com/app/5691)
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365_defender_atp_audit` \
+| dedup host \
+| search OnboardingState!="1" \
+| table host, status, OnboardingState, LastConnected \
+| eval cyences_severity = "high" \
+| `cs_o365_defender_atp_system_is_offboarded_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_o365_defender_atp_system_is_offboarded_filter
+action.cyences_notable_event_action.contributing_events = `cs_o365_defender_atp_audit` | dedup host
+action.cyences_notable_event_action.system_compromised_search = | stats count by host
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_o365_defender_atp_audit` | dedup host | search host=$row.host$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Office 365 Defender ATP
+action.cyences_notable_event_action.teams = SOC, Compliance
+
+
+[Defender ATP - System is not Connected since a Week]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 54 * * * *
+description = This alert notifies when system/device is not connected since a week. \
+\
+Data Collection - Defender ATP Status Check Add-on (https://splunkbase.splunk.com/app/5691)
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365_defender_atp_audit` \
+| dedup host \
+| where LastConnectedEpoch < relative_time(now(), "-1w") \
+| eval is_triggered=0 \
+| append \
+    [| search `cs_cyences_index` alert_name="Defender ATP - System is not Connected since a Week" earliest="-24h@m" \
```

Comment:
Use macro for alert suppression as we might be using for other alerts in the future.

We have one created in SAK App